src/pages/Login/AuthButton.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import Button from "@material-ui/core/Button";
+import ButtonBase from "@material-ui/core/ButtonBase";
+import { makeStyles } from "@material-ui/core/styles";
+import { useAuthContext } from "../../context/AuthContext";
+
+const AuthButton = () => {
+  const history = useHistory();
+  const { user, signout } = useAuthContext();
+
+  const useStyles = makeStyles((theme) => ({
+    large: {
+      width: theme.spacing(8),
+      height: theme.spacing(8),
+    },
+  }));
+
+  const classes = useStyles();
+
+  return user.isAuthenticated ? (
+    <p>
+      Welcome!{" "}
+      <ButtonBase className={classes.image}>
+        <Button
+          href="#"
+          size="small"
+          variant="contained"
+          color="secondary"
+          className={classes.button}
```

Comment:
🔧 Do we need `href` or `className` here?

Reply:
className is needed for the button to display.  href not needed so removed.  Also removed ButtonBase as it was throwing an error.  Commit: d606eb88ec1b9224c41da3188eea5713125bc574